pinot-broker/src/test/java/org/apache/pinot/broker/broker/BrokerManagedAsyncExecutorProviderTest.java

```diff
@@ -49,9 +50,12 @@ public class BrokerManagedAsyncExecutorProviderTest {
 
   @BeforeClass
   public void setUp() {
-    _brokerMetrics = new BrokerMetrics(CommonConstants.Broker.DEFAULT_METRICS_NAME_PREFIX,
-        PinotMetricUtils.getPinotMetricsRegistry(new PinotConfiguration()),
-        CommonConstants.Broker.DEFAULT_ENABLE_TABLE_LEVEL_METRICS, Collections.emptyList());
+    _brokerMetrics = new BrokerMetrics(
+            CommonConstants.Broker.DEFAULT_METRICS_NAME_PREFIX,
```

Comment:
(minor) Please revert this part that doesn't follow [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#intellij)

Reply:
Ahh, I see, it's done and apologies 🙏